### PR TITLE
Update foreman_remote_execution to 4.6.0 and core to 1.5.0

### PR DESCRIPTION
**Pull request by adamruzicka**
- Update foreman_remote_execution to 4.6.0
- Update foreman_remote_execution_core to 1.5.0

Requires:
- [x] https://github.com/theforeman/foreman-packaging/pull/6757

### CODE REVIEW

File: dependencies/buster/foreman_remote_execution_core/control [hunk 1]
```diff
@@ -13,5 +13,5 @@ Package: ruby-foreman-remote-execution-core
 Architecture: all
 XB-Ruby-Versions: ${ruby:Versions}
 Depends: ${shlibs:Depends}, ${misc:Depends}, ruby | ruby-interpreter, rake,
-  ruby-foreman-tasks-core (>= 0.1.5), ruby-net-scp, ruby-net-ssh
+  ruby-foreman-tasks-core (>= 0.1.5), ruby-net-scp, ruby-net-ssh, ruby-smart-proxy-remote-execution-ssh (>= 0.4.0)
```
Comment (evgeni) [inline dependencies/buster/foreman_remote_execution_core/control:16]: shouldn't this change be also in bionic and focal?

**Comment by adamruzicka** [comment]
What's up with the tests? It doesn't feel related

**Comment by evgeni** [comment]
no, but I'll look into it :)

**Comment by evgeni** [comment]
ah yeah, plugins repo for users don't differ by version, so the tasks scratch build for 2.3 you did yesterday, sneaked in and confused everyone, deleting and re-running

**Comment by evgeni** [comment]
[test packaging]

**Review by evgeni — APPROVED** [review]

**Pull request merged**